### ISSUE TITLE
Memory constraint: NOT enforced by Docker in Linux & Mac Kernels

### ISSUE THREAD

**Issue by GastonMazzei** [GastonMazzei/Peat#1]
**Neither of both (RAM & Swap) memory constraints are being enforced in my Linux kernel. It's an open Issue in the Docker Project and it also seems to be an unsolved issue for Mac Kernel.**

**Links:**
(1) https://github.com/docker/for-mac/issues/2931
(2) https://github.com/docker/for-linux/issues/953
(3) https://serverfault.com/questions/762206/cgroups-memory-limit-not-working
Finally: In the Mac Issue (1) they refer to (3) for the solution. PEND. 

**TLDR: solve OR remove the memory-limit functionality. Tho windows users may miss it**
